webui/react/src/e2e/models/components/Page.ts

```diff
@@ -1,4 +1,4 @@
-import { BaseReactFragment } from 'e2e/models/BaseComponent';
+import { BaseReactFragment } from 'e2e/models/base/BaseReactFragment';
 
 /**
  * Represents the PageComponent component in src/components/Page.tsx
```

Comment:
Could this `PageComponent` file be in a `common` or `base` folder too?

Reply:
nope. it's a model that represents the component from https://github.com/determined-ai/determined/blob/main/webui/react/src/components/Page.tsx

Reply:
since they have a Page.tsx, we have a Page.ts model.